src/components/pages/Dashboard/Dashboard.js

```diff
@@ -81,31 +81,33 @@ const Dashboard = props => {
   }, []);
 
   const data = [];
-
+  const escaTickets = tickets.filter(x => x.ticket_status == 'approved');
```

Comment:
I think you should change your double equals to a triple to get rid of the warning.

Reply:
Thanks, this is  better than before.